Sources/MapboxNavigation/DayStyle.swift

```diff
@@ -186,12 +186,14 @@ open class DayStyle: Style {
         ExitView.appearance(for: traitCollection).cornerRadius = 5.0
         ExitView.appearance(for: traitCollection).foregroundColor = .black
         ExitView.appearance(for: traitCollection).borderColor = .black
+        ExitView.appearance(for: traitCollection).highlightColor = .white
```

Comment:
`DayStyle.applyCarPlayManeuversStyling(for traitCollection:) `may also need to be updated.